validation/report.md

```diff
@@ -34,23 +46,28 @@ To run the process and main analysis for the project, the research team extracte
 
 ## Methods
```

Comment:
Given the lit review above, how do our specific methods derive from past work and the state of the art? How do our efforts correspond to and build on what's been done in this broader body of research to date? Lean on David as needed to add a few sentence linking this all together.